### PR TITLE
Jakarta Faces rename

### LINKED ISSUE #24329: Faces 4.0 Specification is now called Jakarta Faces 4.0 not Jakarta Server Faces, update where necessary

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #24329

Latest schema from: https://jakarta.ee/xml/ns/jakartaee/

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

**Comment by posmith** [comment]
L2 message review not required.

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

**Review by isaacrivriv — DISMISSED** [review]
Updates look good to me, thanks!

**Review by volosied — DISMISSED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/resources/l10n/io.openliberty.facesContainer-4.0.properties [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 #ISMESSAGEFILE FALSE
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables CDI integrations for web applications that use the Jakarta Server Faces \
- 4.0 framework. This feature only includes container integration code for Jakarta Server Faces but does not \
- provide any API or implementation. To use this feature, the Jakarta Server Faces interface and implementation \
+description=This feature enables CDI integrations for web applications that use the Jakarta Faces \
+ 4.0 framework. This feature only includes container integration code for Jakarta Faces but does not \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/resources/l10n/io.openliberty.facesContainer-4.0.properties]:
```suggestion
 4.0 framework. This feature includes only container integration code for Jakarta Faces but does not \
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review- minor edit suggestion for misplaced "only"

File: dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/resources/l10n/io.openliberty.facesContainer-4.0.properties (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/resources/l10n/io.openliberty.facesContainer-4.0.properties]: fixed! Thanks for the review!

**Review by dmuelle — APPROVED** [review]
LGTM thanks

**Comment by pnicolucci** [comment]
#libby #build

**Review by tbitonti — APPROVED** [review]

**Pull request merged**